scss/bootstrap/components/_checkbox-switch.scss

```diff
@@ -0,0 +1,153 @@
+/* Variables */
+
+$checkbox-switch-speed:       0.2s;
+$checkbox-switch-gap:         2px;
+$checkbox-switch-size-small:  10px;
+$checkbox-switch-size-normal: 20px;
+$checkbox-switch-size-medium: 30px;
+$checkbox-switch-size-large:  40px;
+
+/* Basic (default) styles */
+
+.checkbox-switch {
+  position: relative;
+}
+
+.checkbox-switch input {
+  bottom: 0;
+  cursor: pointer;
+  display: block;
+  filter: alpha(opacity=0);
+  height: 0%;
+  left: 0;
+  margin: 0;
+  opacity: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 0%;
+}
+
+.checkbox-switch input + span {
+  cursor: pointer;
+  padding-left: $checkbox-switch-size-normal * 2;
+  user-select: none;
+}
+
+.checkbox-switch input + span::before {
```

Comment:
@igorpavlov I understand that the whole styling have been copy-pasted and ironed out a little bit, can you confirm that *every* single line here is actually needed? Any way we can improve the code by removing something?

Reply:
Cleaned.